### PR TITLE
Add formal RibaMetaMod-tests 

### PR DESCRIPTION
**Pull request by HendrikKok**

### CODE REVIEW

File: tests/test_ribametamod_cases.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    mf6_active_drainage_packages = get_mf6_drainage_packagenames(mf6_backwater_model)
+    basin_definition = create_basin_definition(ribasim_backwater_model, buffersize=5.0)
+    mf6_backwater_model = add_rch_package(mf6_backwater_model)
+
+    driver_coupling = DriverCoupling(
+        mf6_model=mf6_modelname,
+        mf6_active_river_packages=mf6_active_river_packages,
+        mf6_active_drainage_packages=mf6_active_drainage_packages,
+    )
+    # should becomde RibaMetaMod-class, including the MetaSWAP model
```
Comment (Hofer-Julian) [inline tests/test_ribametamod_cases.py:66]:
```suggestion
    # TODO: should become RibaMetaMod-class, including the MetaSWAP model
```

File: tests/fixtures/common.py [hunk 2]
```diff
@@ -60,3 +63,15 @@ def create_wells(nrow: int, ncol: int, idomain: xr.DataArray) -> mf6.WellDisStru
...
+def create_wells_max_layer(
+    nrow: int, ncol: int, idomain: xr.DataArray
+) -> mf6.WellDisStructured:
+    """
+    Create wells, deactivate inactive cells. This function wouldn't be necessary
+    if iMOD Python had a package to specify wells based on grids.
+    """
+
+    wel_layer = idomain.layer.max().item()
+    return create_wells(nrow, ncol, idomain, wel_layer)
```
Comment (Hofer-Julian) [inline tests/fixtures/common.py:77]: Maybe something to upstream to iMOD Python? @Huite

File: tests/test_ribametamod.py [hunk 3]
```diff
@@ -1,400 +1,127 @@
...
+    )  # the RibaMetaMod should do this eventually
 
-    write_metamod_toml(
-        path_dev,
-        modflow6_dll,
-        modflow6_model_dir,
-        metaswap_dll,
-        metaswap_dll_dependency,
-        metaswap_model_dir,
+    subprocess.run(
```
Comment (Hofer-Julian) [inline tests/test_ribametamod.py:34]: Please use the function `run_coupler_function` instead

File: tests/test_ribametamod.py [hunk 4]
```diff
@@ -1,400 +1,127 @@
...
+    ribamod_model.write(
+        tmp_path_dev,
+        modflow6_dll=modflow_dll_devel,
+        ribasim_dll=ribasim_dll_devel,
+        ribasim_dll_dependency=ribasim_dll_dep_dir_devel,
+    )
+    msw_model.write(
+        tmp_path_dev / "metaswap"
+    )  # the RibaMetaMod should do this eventually
+    subprocess.run(
```
Comment (Hofer-Julian) [inline tests/test_ribametamod.py:62]: Please use `run_coupler_function` instead.

**Review by Hofer-Julian — COMMENTED** [review]
I had a few small comments

File: tests/fixtures/common.py (hunk 2 above)
Comment (HendrikKok) [reply tests/fixtures/common.py]: This docstring is maybe a bit confusing. I copied it from the existing 'create_wells'. I only extended the functionality so it does not always use layer 3 explicitly. I updated the docstring.

**Review by rleander — APPROVED** [review]
Looks ok, will be tested anyway when setting up the test_ribametamod

**Pull request merged**